src/Illuminate/Mail/Transport/MandrillTransport.php

```diff
@@ -54,10 +63,8 @@ public function stop()
 	 */
 	public function send(Swift_Mime_Message $message, &$failedRecipients = null)
 	{
-		$client = $this->getHttpClient();
-
-		return $client->post('https://mandrillapp.com/api/1.0/messages/send-raw.json', [
-			'body' => [
+		return $this->client->post('https://mandrillapp.com/api/1.0/messages/send-raw.json', [
```

Comment:
You can check `$client::VERSION` here and either use the v5 or v6 args to support both versions.